README.md

```diff
@@ -322,7 +322,7 @@ The Chrome DevTools MCP server supports the following configuration option:
 <!-- BEGIN AUTO GENERATED OPTIONS -->
 
 - **`--browserUrl`, `-u`**
-  Connect to a running Chrome instance using port forwarding. For more details see: https://developer.chrome.com/docs/devtools/remote-debugging/local-server.
```

Comment:
Aside: Does anyone know why  this Case 1 https://developer.chrome.com/docs/devtools/remote-debugging/local-server#port-forwarding-instance is documented?  The android port forwarding bit makes sense but  I don't know why you'd use one Chrome to debug another that's also running on the same machine.  (And I can't come up with good usecase for cross-LAN debugging but.. if this is for it.. it's not explained appropriately).  Asking because if nobody has a defense, I'll put up a CL to drop it.